query/src/common/hashtable/hash_table.rs

```diff
@@ -55,7 +57,7 @@ impl<Key: HashTableKeyable, Entity: HashTableEntity<Key>> Drop for HashTable<Key
 
 impl<Key: HashTableKeyable, Entity: HashTableEntity<Key>> HashTable<Key, Entity> {
     pub fn create() -> HashTable<Key, Entity> {
-        let size = (1 << 8) * mem::size_of::<Entity>();
+        let size = (1 << 12) * mem::size_of::<Entity>();
```

Comment:
Why need change from 8 to 12?

Reply:
It's test code. I change the bit size to observer the proportion of resize. It wll be 8 finally.